app/src/main/java/com/duckduckgo/app/global/model/SiteMonitor.kt

```diff
@@ -163,6 +165,7 @@ class SiteMonitor(
 
     override fun privacyProtection(): PrivacyShield {
         userAllowList = domain?.let { isAllowListed(it) } ?: false
+        if (duckPlayer.isDuckPlayerUri(url)) return UNKNOWN
```

Comment:
is this to avoid showing the unprotected red dot?

Reply:
Yes. Shouldn't be needed as we generally show either one or the other, but at the same time I figured we could skip executing this logic as we won't use it